### ISSUE TITLE
Logs not visible when running the application

### ISSUE THREAD

**Issue by sw-tracker** [gridatek/jnxplus#909]
Hi there,

I have the following under targets:

```
"serve": {
      "dependsOn": ["maven-serve"]
    },
    "maven-serve": {
      "options": {
        "task": "spring-boot:run -Dspring-boot.run.arguments=\"--spring.profiles.active=local\""
      }
    },
```

When I run my application on its own, the logs are shown on the console:

`nx run my-app:serve`

But when I run multiple apps, I cannot see the logs from multiple apps:

`nx run-many --parallel=8 --verbose=true --target=serve`

How can I see the logs from all spring boot applications?

**Comment by khalilou88** [comment]
Hi @sw-tracker 
Thanks for raising this issue.
I will try to reproduce this bug with this repo : https://github.com/khalilou88/jnxplus-examples
I don't know how to solve it but thinking about changing `execSync` by `spawnsync`

**Comment by sw-tracker** [comment]
Thanks @khalilou88 . I was thinking that it would be cool if each app showed a prefix in each of the logs, that way when multiple apps run in parallel we can identify to which app the log belongs to. Something similar to the way [concurrently](https://www.npmjs.com/package/concurrently) does it.

Here is a small example of how 4 apps running in parallel using concurrently look like:
![image](https://github.com/khalilou88/jnxplus/assets/15125668/eb1f705b-b68e-4b50-a338-7fc572a89a56)

Let me know if you need any help reproducing the issue.

**Comment by sw-tracker** [comment]
Hi @khalilou88 ,

In our pipelines, it would suffice being able to forward all the logs to a file that we can view when evaluating pipeline errors.

**Comment by khalilou88** [comment]
@sw-tracker 
You are asking for nice features, but I don't know how to make them and also don't have much time.
I will accept new PR for that.
A new executor in experimental mode will be good, then merge it with the run-task executor later.
